pkgs/applications/graphics/paraview/default.nix

```diff
@@ -38,6 +35,7 @@ mkDerivation rec {
   nativeBuildInputs = [
     cmake
     makeWrapper
+    wrapQtAppsHook
```

Comment:
This already uses the specific qt deriver `mkDerivation` which automatically adds this, plus it's not in the docs to do this explicitly. Can you drop this?

Reply:
Sure!